OpenApiSpecs/exchangeSetService_OpenApi_definition_internal.yaml

```diff
@@ -181,13 +323,60 @@ paths:
               schema:
                 $ref: "#/components/schemas/exchangeSetResponse"
 
-        "304":
-          description: Not modified.
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/errorDescription"
+
+        "401":
+          $ref: "#/components/responses/unauthorised"
+
+        "403":
+          $ref: "#/components/responses/forbidden"
+
+        "429":
+          $ref: "#/components/responses/tooManyRequests"
+
+        "500":
+          $ref: "#/components/responses/internalServerError"
+
+      callbacks:
+        fulfilmentResponse:
+          $ref: "#/components/callbacks/fulfilmentResponse"
+          
+  /v2/exchangeSet/s100/updatesSince:
+    post:
+      summary: Provide all the releasable data after a datetime.
+      description: |
+        Given a datetime, build an Exchange Set of all the releasable Product versions that have been issued since that datetime.
+        
+      operationId: getS100Products
+
+      tags:
+         - s100
+         
+      parameters:
+        - $ref: "#/components/parameters/sinceDateTime"
```

Comment:
Shall `sinceDateTime` and `productIdentifier` be request body parameters?

Reply:
This operation is a post operation to keep it consistent with the productNames and productVersions endpoint, but otherwise has no body. It may feel more consistent to have the sinceDateTime parameter in the body. Question, should it be the whole body in RFC 1123 format (i.e. `Wed, 21 Oct 2015 07:28:00 GMT`), or as a part of a JSON payload (.e. 
````json
{ "sinceDateTime":"2015-10-21T07:28:00.000Z"}
```

My feeling is as part of a JSON object, or as is in a RFC 1123 format query parameter, not as a RFC 1123 body.

Reply:
I think I'd prefer a JSON Body (i.e. `{ "sinceDateTime":"2015-10-21T07:28:00.000Z"}` ) because it isn't an optional parameter. The other query params are optional.  The body defines the subset of the service you want, the query params are options on how you want it delivered (i.e. callback).

Reply:
I agree, a JSON body is better. It also provides flexibility for any additional new property needed in future.

Reply:
Agree, JSON body is better option and make it extensible for other properties.

Reply:
sinceDateTime added in the JSON Body.